### DISCUSSION TITLE
Security Review?

### DISCUSSION THREAD

**Discussion by codeninja75** [minio/minio#13446] [Q&A]
Is there a 3rd party security review of min.io's codebase?

**Comment by harshavardhana** [comment]
> Is there a 3rd party security review of min.io's codebase?

what for?

**Comment by codeninja75** [comment]
It's a requirement the company has for putting software in our network.  https://www.cncf.io/blog/2019/08/06/open-sourcing-the-kubernetes-security-audit/ as an example of what our security folks like to read for fun.

**Reply by klauspost** [reply]
It is not a requirement we have encountered from our current customers. Please write to`sales@minio.io` if you want to discuss your requirements.
